assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -125,8 +124,9 @@ const renderEdit = ({ tabs }, setAttributes, updateTabAttribute) => {
     <InspectorControls>
       <PanelBody title={__('Settings', 'planet4-blocks-backend')}>
         {tabs.map((tab, index) => {
-          const { button } = tab;
+          const {button} = tab;
           if (!button) {
+            // eslint-disable-next-line array-callback-return
             return;
```

Comment:
Same as before, maybe if you return `null` you'll be avoided to add `array-callback-return`.

Reply:
Yea, same to as with the code above, this is just a workaround to prevent the editor from crashing. Returning `null` may actually cause that.